.github/workflows/ci.yml

```
@@ -1,16 +1,14 @@
-name: Lint, test, and publish
+name: CI
 
 on:
   push:
     branches:
-      - develop
-  release:
-    types: [published]
+      - main
+  pull_request:
```

```suggestion
  pull_request:
    branches: [main]
```

That way we only run this if there is a PR to `main`. Not sure what we want